_data/simple-icons.json

```diff
@@ -3849,6 +3849,12 @@
             "hex": "FC5531",
             "source": "https://www.csdn.net/company/index.html"
         },
+        {
+            "title": "CSS",
+            "hex": "663399",
+            "source": "https://github.com/CSS-Next/logo.css/blob/4e5bae2d287e5523dc30cbdd8d9f4885b307adc7/css.svg",
```

Comment:
```suggestion
            "source": "https://github.com/CSS-Next/logo.css/blob/bacc20878227204b283c68a6b935f8279e06b0cd/css.svg",
```
Source changed. Attaching the wire frame of the comparison as a screenshot.
![image](https://github.com/user-attachments/assets/7707099c-0931-469a-925f-54846638774b)